src/eckit/web/HttpResource.cc

```diff
@@ -229,14 +229,11 @@ void HttpResource::dispatch(eckit::Stream&, std::istream&, HttpStream& out, Url&
 
                 std::ostringstream oss;
                 oss << "Unsupported HTTP method " << method << " url=" << url;
-                throw eckit::MethodNotYetImplemented(oss.str());
+                throw eckit::NotImplemented(oss.str());
```

Comment:
We should deprecate the replaced exception (MethodNotYetImplemented) as there is no code in eckit supporting this exception anymore and none of these projects: atlas, atlas-orca, bufrdc_tables, dhskit, eccodes, eckit, fckit, fdb, fdb5, mars-adsm, mars-client, mars-hpss, mars-server, metkit, mir, multio, odc, pgen, transi, ...

Reply:
OK, I don't know the history of this one. If there is no occurrence anywhere in the stack we can just remove it rather than deprecate?
If no occurrence anymore, which projects would have used it in the past, and how long ago?

Reply:
Deprecated with commit [8510f4e](https://github.com/ecmwf/eckit/pull/110/commits/8510f4e9e66dd1f3bcdcd424738baf2dd7d07ea2)